java/code/src/com/redhat/rhn/domain/rhnpackage/Package.java

```diff
@@ -100,6 +100,14 @@ public Boolean isLockPending() {
         return lockPending;
     }
 
+    /** Check if the package is part of a retracted patch
+     *
+     * @return true if the package is part of a retracted patch
+     */
+    public Boolean isPartOfRetractedPatch() {
+        return errata.stream().anyMatch(e -> "retracted".equals(e.getAdvisoryStatus()));
```

Comment:
Is there a reason why the enum is not used here?

Reply:
this seems like something the automatic refactoring to the enum did not catch.